src/components/widgets/index.js

```diff
@@ -29,7 +29,7 @@ const widgets = [
   BookSelectWidget,
   InterlinearWidget,
   PassageWidget,
-  SelectablePassageWidget,
+  SelectablePassageWidget
```

Comment:
@mjhea0 I do prefer trailing commas though.  Anyway we can configure that as something that doesn't get cleaned up and in fact enforced?

Reply:
Yes, fixed this.